app/src/main/java/com/davidtomas/taskyapp/features/agenda/presentation/AgendaDetail/AgendaDetailScreen.kt

```diff
@@ -0,0 +1,265 @@
+                    text = "Visitors"
+                )
+                Spacer(modifier = Modifier.height(17.dp))
+                PillsSelector(
+                    modifier = Modifier.fillMaxWidth(),
+                    onPillClicked = {}
+                )
+                Spacer(modifier = Modifier.height(17.dp))
+                Text(
+                    style = MaterialTheme.typography.titleSmall,
+                    text = "Going"
+                )
+                Spacer(modifier = Modifier.height(12.dp))
+                Column(
+                    verticalArrangement = Arrangement.spacedBy(5.dp)
+                ) {
+                    AttendeeItem(
+                        fullName = "Ann Allen",
+                        onDeleteIconClicked = { /*TODO*/ },
+                        isEditable = isEditable
+                    )
+                    AttendeeItem(
+                        fullName = "Wade Warren",
+                        onDeleteIconClicked = { /*TODO*/ },
+                        isEditable = isEditable
+                    )
+                    AttendeeItem(
+                        fullName = "Esther Howard",
+                        onDeleteIconClicked = { /*TODO*/ },
+                        isEditable = isEditable
+                    )
+                }
+                Spacer(modifier = Modifier.height(17.dp))
+                Text(
+                    style = MaterialTheme.typography.titleSmall,
+                    text = "Not Going"
+                )
+                Spacer(modifier = Modifier.height(12.dp))
+                Column(
+                    verticalArrangement = Arrangement.spacedBy(5.dp)
+                ) {
+                    AttendeeItem(
+                        fullName = "Ann Allen",
+                        onDeleteIconClicked = { /*TODO*/ },
+                        isEditable = isEditable
+                    )
+                    AttendeeItem(
+                        fullName = "Wade Warren",
+                        onDeleteIconClicked = { /*TODO*/ },
+                        isEditable = isEditable
+                    )
+                    AttendeeItem(
+                        fullName = "Esther Howard",
+                        onDeleteIconClicked = { /*TODO*/ },
+                        isEditable = isEditable
+                    )
+                }
+                Spacer(modifier = Modifier.height(25.dp))
+            }
+            Box(
+                modifier = if (agendaType != AgendaType.EVENT) Modifier.weight(1f) else Modifier,
+                contentAlignment = Alignment.BottomCenter
+            ) {
+                Button(
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    onClick = { /*TODO*/ }
+                ) {
+                    Text(text = "Delete Item")
+                }
+            }
```

Comment:
Could be put in a separate composable as well to not bloat this detail composable that much with a conditional section